project/src/components/offer-card/offer-card.tsx

```diff
@@ -1,18 +1,33 @@
-function PlaceCard(cardKey: number): JSX.Element{
+import {Offer} from '../../types/offer';
+import {MouseEvent} from 'react';
+import {Link} from 'react-router-dom';
+
+function OfferCard(offer: Offer, setIdActiveOffer : (a: number) => void): JSX.Element{
```

Comment:
```suggestion
function OfferCard(props): JSX.Element{
```
нене, это должно быть компонент же